src/mixins/filing-template-mixin.ts

```diff
@@ -218,21 +222,11 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         legalName: this.getBusinessSnapshot.businessInfo.legalName
       },
       alteration: {
-        provisionsRemoved: this.getProvisionsRemoved,
         business: {
           identifier: this.getBusinessId,
           legalType: this.getEntityType
         },
-        nameRequest: {
-          legalType: this.getNameRequest.legalType,
-          legalName: this.getNameRequest.legalName,
-          nrNumber: this.getNameRequest.nrNumber
-        },
-        nameTranslations: nameTranslations,
-        shareStructure: {
-          resolutionDates: this.getNewResolutionDates,
-          shareClasses
-        },
+        provisionsRemoved: this.getProvisionsRemoved,
         contactPoint: {
           email: this.getBusinessContact.email,
           phone: this.getBusinessContact.phone,
```

Comment:
It's not related to this change but can you please test if the `contactPoint.extension` can be null here? I changed it in the create-ui. Maybe the same change can/should be done here.

Reply:
Good call, i will have a look!
I bumped into the same thing in the ShareStructure but noticed you had already fixed it in the Shared Component, Gj! 👍

Reply:
Yessir, no issues here. It will pass as undefined if there is no Extension.

Reply:
Tried without a phone number as well, since it's pulled from the account and could potentially be null. All good though 👍

Reply:
Thanks!